src/microbe_stage/ChunkConfiguration.cs

```diff
@@ -146,6 +146,8 @@ public class ChunkScene
         /// </summary>
         public string ScenePath;
```

Comment:
This and the loaded scene have to be removed in order to not cause confusion by having 2 separate mechanisms at once specifying how the visual is defined.

Reply:
This will cause to at least chunks on microbe death not function. Last time I tried to do it with fixing said issue, it froze my game.

Reply:
Those chunks must also be updated to the new system. I won't have the confusion of having 2 systems for cell chunk visuals.